plans/example-browser-node/compositions/sync-cross-runtime.toml

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name    = "sync-cross-runtime"
+author  = "Glen De Cauwsemaecker"
+
+[global]
+plan    = "example-browser-node"
+case    = "sync"
+builder = "docker:generic"
+runner  = "local:docker"
+
+total_instances = 4
```

Comment:
For what it's worth, if you don't use percentages in your groups, this field is optional since Testground can compute the value.

Reply:
Addressed it for the sake of completness.